packages/itwinui-css/src/input-container/input-container.scss

```diff
@@ -104,6 +104,13 @@
     &:is(:not(:first-child)) {
       padding-inline-start: var(--iui-size-2xs);
     }
+
+    &::placeholder {
+      color: var(--iui-color-text-placeholder);
+      font-weight: var(--iui-font-weight-light);
+      opacity: 1;
+      user-select: none;
```

Comment:
is `user-select: none` needed?

Reply:
Probably not, but it's added to the inputs so kept it here.

Reply:
looks like it wasn't added in #1485, it was already present before.
<img width="779" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/dad1eaf7-3b07-449c-bc3e-7b75d3f09318">

if it's not needed, then let's not add it here in searchbox

Reply:
removed from SearchBox